frappe/utils/html_utils.py

```diff
@@ -42,7 +42,7 @@ def clean_html(html):
 			"td",
 			"tr",
 		],
-		attributes=[],
+		attributes=["data-list"],
```

Comment:
Minor problem with this, this is overriding default allowed attributes. 

reference: https://bleach.readthedocs.io/en/latest/clean.html#allowed-attributes-attributes